ui/packages/catalog-ui-search/src/main/webapp/component/visualization/maps/openlayers/map.openlayers.js

```diff
@@ -202,6 +203,21 @@ module.exports = function OpenlayersMap(
     onCameraMoveEnd: function(callback) {
       map.on('moveend', callback)
     },
+    doPanZoom: function(coords) {
+      const that = this
+      that.zoomOut({ duration: 1000 }, () => {
+        setTimeout(() => {
+          that.zoomToExtent(coords, { duration: 2000 })
+        }, 0)
+      })
+    },
+    zoomOut: function(opts, next) {
+      if (map.getView().getZoom() > 6) {
```

Comment:
What is this 6 constant?

Reply:
To keep the pan/zoom transition from being too jarring, when the camera is zoomed in close enough we first execute a zoom out before zooming to the targeted location.